ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -258,6 +268,18 @@
         3,
         Importance.MEDIUM,
         KSQL_HEARTBEAT_THREAD_POOL_SIZE_CONFIG_DOC
+    ).define(
+        KSQL_LAG_REPORTING_ENABLE_CONFIG,
+        Type.BOOLEAN,
+        false,
+        Importance.MEDIUM,
+        KSQL_LAG_REPORTING_ENABLE_DOC
+    ).define(
+        KSQL_LAG_REPORTING_SEND_INTERVAL_MS_CONFIG,
+        Type.LONG,
+        1000L,
```

Comment:
can we make this something like 3 or 5 seconds. I know it sounds conservative.. but since this is RPC on kafka, best to err on that side

Reply:
Sure.  Choosing 5 seconds